packages/lab/src/pill/PillNext.css

```diff
@@ -0,0 +1,146 @@
+/* Styles applied to the root element */
+.saltPill {
```

Comment:
if you're going to keep Pill.css, i would rename this PillNext for now otherwise you'll probably notice clashes
since you're not even exporting Pill anymore, is it worth even having this as Next?

Reply:
No, I will delete all of the old Pill code. The PillNext is just so reviewers don't have to deal with the diff.